shop-local/src/App.css

```diff
@@ -13,26 +13,39 @@
   }
 }
 
-.App-header {
+.header {
   background-color: #282c34;
-  min-height: 100vh;
+  min-height: 5vh;
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
   align-items: center;
-  justify-content: center;
+  justify-content: space-between;
   font-size: calc(10px + 2vmin);
   color: white;
 }
 
-.App-link {
-  color: #61dafb;
+.StoreInfo{
```

Comment:
I think to keep with the a style guide we should use hyphens to separate words instead of camelCase. So in this case it would be store-info